var/spack/repos/builtin/packages/elpa/package.py

```diff
@@ -32,6 +32,7 @@ class Elpa(AutotoolsPackage):
     homepage = 'http://elpa.mpcdf.mpg.de/'
     url = 'http://elpa.mpcdf.mpg.de/elpa-2015.11.001.tar.gz'
 
+    version('2016.11.001.pre', '5656fd066cf0dcd071dbcaf20a639b37')
```

Comment:
Is this a pre-release? We may want to mark the last stable release as `preferred`.